From e4f3035da324bc561619ae634d2dc9651a400fa5 Mon Sep 17 00:00:00 2001
From: Bharat B Panda <bharat.b.panda@intel.com>
Date: Thu, 21 Jul 2022 15:03:42 +0530
Subject: [PATCH] Add Intel provided p2p interface name

CtsVerifier GO test was failing as it was unable to tether over
p2p interface, the root cause was intel provided p2p interface
name does not sub-match with the google provided iface name.

From Android 12 onwards, the wifi and connectivity framework
has been moved to mainline module as an apex package hence the
config value it will pick from respective package configs.

Added iface name entry to support intel provide p2p iface name.

Signed-off-by: Bharat B Panda <bharat.b.panda@intel.com>
---
 Tethering/res/values/config.xml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Tethering/res/values/config.xml b/Tethering/res/values/config.xml
index 0412a49e2..85a359f63 100644
--- a/Tethering/res/values/config.xml
+++ b/Tethering/res/values/config.xml
@@ -55,6 +55,7 @@
          Wifi P2P interfaces.  If the device doesn't want to support tethering over Wifi P2p this
          should be empty.  An example would be "p2p-p2p\\d-.*" -->
     <string-array translatable="false" name="config_tether_wifi_p2p_regexs">
+        <item>"p2p-wlan\\d-.*"</item>
         <item>"p2p-p2p\\d-.*"</item>
         <item>"p2p\\d"</item>
     </string-array>
-- 
2.37.0

